### ISSUE TITLE
Record and replay user input (macro)

### ISSUE THREAD

**Issue by yume-chan** [yume-chan/ya-webadb#487]
Leart from https://github.com/Genymobile/scrcpy/issues/2312 that it's possible to get all user inputs using only shell permission. It also outputs in a machine-readable format.

So it's possible to create an App that records user inputs using `getevent`, then playback using Scrcpy (or `input`), which will be useful for automation.

**Comment by mattwakeman** [comment]
Love the idea of this

**Comment by icelingg** [comment]
That's correct. I like this operation too. I've already implemented it in my application, but it only works for upper‑level input. It would be even better if touch‑level input could be achieved.

**Comment by yume-chan** [comment]
What's upper-level input and touch-level input? Doesn't `getevents` log all inputs and you can inject them using Scrcpy protocol?
